plugins/google_admin/plugin.spec.yaml

```
@@ -1,142 +1,189 @@
 plugin_spec_version: v2
 extension: plugin
-products: [insightconnect]
+products: [ insightconnect ]
 name: google_admin
 title: Google Apps Admin
 description: Get and suspend users with the Google admin plugin
-version: 1.0.3
+version: 1.0.4
+supported_versions: ["12/07/2021"]
```

Thanks to an update by Adam this morning, we can now do away with the quotes here.
```suggestion
supported_versions: [12/07/2021]
```